crates/core/src/operations/create.rs

```diff
@@ -20,7 +20,7 @@ use crate::logstore::LogStoreRef;
 use crate::protocol::{DeltaOperation, SaveMode};
 use crate::table::builder::ensure_table_uri;
 use crate::table::config::TableProperty;
-use crate::table::TableParquetOptions;
+use crate::table::table_parquet_options::FileFormatOptions;
```

Comment:
I went with the trait name `FileFormatOptions`. It's perhaps a bit vague, but I want it to suggest that this could apply to underlying file types beyond parquet. Also, I couldn't think of a better name.

Reply:
It probably makes sense to rename the `table_parquet_options` module too then, maybe we could call it `format`?

Reply:
Done.